src/shared/AppServicesDeleteModal/AppServicesDeleteModal.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import React, { useState } from "react";
+
+import { AppServicesDeleteModal } from "./AppServicesDeleteModal";
+
+export default {
+  title: "Components/Shared/DeleteModal",
+  component: AppServicesDeleteModal,
+  args: {
+    title: "sample title",
+    selectedItemData: "name of the element to delete",
+  },
+} as ComponentMeta<typeof AppServicesDeleteModal>;
+
+const Template: ComponentStory<typeof AppServicesDeleteModal> = (
+  args,
+  { id }
+) => {
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  return (
+    <div
+      style={{ transform: "scale(1)", minHeight: 400, height: "100%" }}
+      id={id}
+    >
+      <AppServicesDeleteModal
+        {...args}
+        isModalOpen={isModalOpen}
+        getModalAppendTo={() => document.getElementById(id)}
+        handleModalToggle={() => setIsModalOpen(false)}
+      />
+      <button onClick={() => setIsModalOpen(true)}>Open modal</button>
+    </div>
```

Comment:
@suyash-naithani check how I handled the modal here, I think we have to have them start closed and then open manually for them to work peacefully with Storybook.

Reply:
I have actually expanded this quite a bit from when I pinged you. Please check the latest version.